### PR TITLE
Mise-à-jour de mon profil BBL

### PR DESCRIPTION
**Pull request by borisschapira**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -4860,11 +4860,11 @@ var data = {
             "since": "2014-02-14",
             "name": "Boris Schapira",
             "bio": "Consultant - Clever Age",
-            "picture": "http://avatars.borisschapira.com/SudWeb12_300.jpg",
+            "picture": "//avatars.borisschapira.com/boris_1light_300_fuchsia.jpg",
```
Comment (dadoonet) [inline baggers.js:4863]: Http missing, no?

File: baggers.js [hunk 2]
```diff
@@ -4881,28 +4881,32 @@ var data = {
...
-                    "summary": "Nous avons tous des idées. Petites, grandes, incongrues, logiques. Comment les capturer, les formaliser, les confronter pour transformer ces idées en projet ? Introduction à Lean Canvas et aux outils du Service Design."
+                    "title": "Pas de fatalité, il suffit de s'impliquer",
+                    "summary": "Quand on est salarié, on peut avoir l'impression que l'entreprise dans laquelle on travaille est un agglomérat complexe d'entités juridiques, administratives, de production, de contrôle… Parfois, certains se sentent frustrés de ne pas pouvoir changer les choses. Pourtant, de nombreuses implications sont possibles pour apporter à l'entreprise un peu d'humanité."
                 },
                 {
-                    "title": "Rappels sur JavaScript",
-                    "summary": "JavaScript est un langage manipulé par de nombreuses professions du Web, sans pour autant toujours en connaitre les secrets. Petit tour d'horizon du langage, des bonnes pratiques, des pièges à éviter et des librairies & frameworks à connaitre."
+                    "title": "Mon pire client a 5 ans",
```
Comment (dadoonet) [inline baggers.js]: Doublon?

**Comment by dadoonet** [comment]
Left some comments

File: baggers.js (hunk 1 above)
Comment (borisschapira) [inline baggers.js:4863]: Nope : if you do not precise it, the browser will use the current protocol. As I support both http and https on avatars.borisschapira.com, it ensure that my avatar will be loaded in https if the website, in the future, migrate to https.

Comment (dadoonet) [inline baggers.js:4863]: Makes sense. Thanks.

**Pull request merged**